warehouse/templates/email/added-as-collaborator.body.txt

```diff
@@ -1,4 +1,17 @@
-You have been added as {{ role }} to the PyPI project {{ project }} by {{ submitter }}.
+You have been added as a collaborator to the PyPI project {{ project }}
+by {{ submitter }}.
+
+{% if role == 'Owner' %}
+You are now a project owner. This means that you may add other collaborators,
+upload releases for the project, delete files and releases, or delete the entire
+project.
+{% endif %}
+
+{% if role == 'Maintainer' %}
+You are now a project maintainer. This means that you may upload releases for
+the project. You cannot add collaborators, delete files, delete releases, or
+delete the project.
+{% endif %}
```

Comment:
These conditionals are going to produce a lot of extra newlines in this template. We probably want to remove the newline in the middle, and also use `{%-` and `-%}` to strip the newlines that the conditional statements are on.

You can see exactly what an email will look like in the terminal in which you ran `make serve` whenever one is sent.